src/components/checkSelect.tsx

```diff
@@ -68,8 +68,11 @@ const CheckSelect: React.FC<CheckSelectProps> = ({
                 </svg>
               </div>
             </div>
-            <div className="text-body-small">
-              <label htmlFor="comments" className="text-primary-text">
+            <div className="text-sm/6">
+              <label
+                htmlFor={checkSelection.id}
+                className="font-medium text-gray-900"
```

Comment:
These tailwind classes should use the system design presets that are defined in tailwind.config.ts

For dark text, use `text-primary-text`. For greyish text, use `text-secondary-text`

In this case, I concur you should use `text-primary-text`